frontend/views/containers/access/LoginForm.vue

```diff
@@ -104,7 +112,13 @@ export default ({
     form: {
       username: {
         [L('A username is required.')]: required,
-        [L('A username cannot contain whitespace.')]: noWhitespace
+        [L('A username cannot contain whitespace.')]: noWhitespace,
```

Comment:
For all of the validations that this has in common with `SignupForm.vue`, could you store the on an object in `SignupForm.vue` that you import here?

e.g. in `SignupForm.vue`:

```js
export const signupValidations = {
  [L('A username is required.')]: required,
  // ...
}
```

Then here:

```js
import { signupValidations } from ...
// ...
username: {
  ...signupValidations,
  // ...
}
```